packages/component-library/src/components/plmg-svg-icon/icon.enum.ts

```diff
@@ -0,0 +1,2077 @@
+export enum ICON {
+  Abc = 'abc',
+  AcUnit = 'acUnit',
+  AccessAlarm = 'accessAlarm',
+  AccessAlarms = 'accessAlarms',
+  AccessTime = 'accessTime',
+  AccessTimeFilled = 'accessTimeFilled',
+  Accessibility = 'accessibility',
+  AccessibilityNew = 'accessibilityNew',
+  Accessible = 'accessible',
+  AccessibleForward = 'accessibleForward',
+  AccountBalance = 'accountBalance',
+  AccountBalanceWallet = 'accountBalanceWallet',
+  AccountBox = 'accountBox',
+  AccountCircle = 'accountCircle',
+  AccountTree = 'accountTree',
+  AdUnits = 'adUnits',
+  Adb = 'adb',
+  Add = 'add',
+  AddAPhoto = 'addAPhoto',
+  AddAlarm = 'addAlarm',
+  AddAlert = 'addAlert',
+  AddBox = 'addBox',
+  AddBusiness = 'addBusiness',
+  AddCard = 'addCard',
+  AddChart = 'addChart',
+  AddCircle = 'addCircle',
+  AddCircleOutline = 'addCircleOutline',
```

Comment:
There is 2 versions of this icon: "normal" and "outline". I believed we only wanted the outline icons.